tests/test_buffer_protocol.rs

```diff
@@ -112,7 +112,7 @@ fn test_buffer_referenced() {
         let input = vec![b' ', b'2', b'3'];
         let gil = Python::acquire_gil();
         let py = gil.python();
-        let instance: PyObject = TestBufferClass {
+        let instance = TestBufferClass {
```

Comment:
Users migrating might have to remove a number of `: PyObject` annotations like this.